web-app/views/game.hbs

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8" {{#needToWaitForOpponent readyToPlay currentPlayerTurnToPlay}}http-equiv="refresh" content="5"
+    <meta charset="utf-8" {{#needToWaitForOpponent readyToPlay currentPlayerTurnToPlay}}http-equiv="refresh" content="3"
```

Comment:
As a memo, don't forget to set it lower(1 as in the mobile app?)

Reply:
I'll try with 1 and see how it looks 🙂